.github/workflows/master.yml

```diff
@@ -105,7 +105,7 @@ jobs:
         java-version: '11'
     - name: install R
       run: |
-        echo 'deb https://cloud.r-project.org/bin/linux/ubuntu bionic-cran35/' | sudo tee -a /etc/apt/sources.list
+        echo 'deb https://cloud.r-project.org/bin/linux/ubuntu bionic-cran40/' | sudo tee -a /etc/apt/sources.list
```

Comment:
Let's ignore this for now. I will track it in SPARK-31587

Reply:
I pushed before I saw your note. Will leave this until that issues fixed & we can update all the other PRs